components/dashboard/src/teams/Members.tsx

```diff
@@ -29,29 +34,105 @@ export default function() {
+            </div>
+            <ItemsList className="mt-2">
+                <Item header={true} className="grid grid-cols-3">
+                    <ItemField>
+                        <span className="pl-14">Name</span>
+                    </ItemField>
+                    <ItemField>
+                        <span>Joined</span>
+                    </ItemField>
+                    <ItemField className="flex items-center">
+                        <span className="flex-grow">Role</span>
+                        <ItemFieldContextMenu />
+                    </ItemField>
+                </Item>
+                {members.map(m => <Item className="grid grid-cols-3">
+                    <ItemField className="flex items-center">
+                        <div className="w-14">{m.avatarUrl && <img className="rounded-full w-8 h-8" src={m.avatarUrl || ''} alt={m.fullName} />}</div>
+                        <div>
+                            <div className="text-base text-gray-900 dark:text-gray-50 font-medium">{m.fullName}</div>
+                            <p>{m.primaryEmail}</p>
+                        </div>
+                    </ItemField>
+                    <ItemField>
+                        <span className="text-gray-400">{moment(m.memberSince).fromNow()}</span>
+                    </ItemField>
+                    <ItemField className="flex items-center">
+                        <span className="text-gray-400 flex-grow capitalize">{m.role}</span>
+                        <ItemFieldContextMenu menuEntries={[
+                            {
+                                title: 'Remove',
+                                customFontStyle: 'text-red-600 dark:text-red-400 hover:text-red-800 dark:hover:text-red-300',
+                                onClick: () => { /* TODO(janx) */ }
+                            },
+                        ]} />
+                    </ItemField>
+                </Item>)}
+            </ItemsList>
         </div>
+        {showInviteModal && <Modal visible={true} onClose={() => setShowInviteModal(false)}>
+            <h3 className="mb-4">Invite Members</h3>
+            <div className="border-t border-b border-gray-200 dark:border-gray-800 -mx-6 px-6 py-4 flex flex-col">
+                <label htmlFor="inviteUrl" className="font-medium">Invite URL</label>
+                <div className="w-full relative">
+                    <input name="inviteUrl" disabled={true} readOnly={true} type="text" value={getInviteURL()} className="rounded-md w-full truncate pr-8" />
+                    <div className="cursor-pointer" onClick={() => copyToClipboard(getInviteURL())}>
+                        <img src={copy} title="Copy Invite URL" className="absolute top-1/3 right-3" />
+                    </div>
+                </div>
+                <p className="mt-1 text-gray-500 text-sm">{copied ? 'Copied to clipboard!' : 'Use this URL to join this team as a Member.'}</p>
```

Comment:
**praise**: Nice addition!

**suggestion**: Ideally this could use a tooltip as seen in the specs that changes the tooltip text on click from _Copy URL_ to _Copied!_.

Reply:
Fixed in https://github.com/gitpod-io/gitpod/pull/4571 🙌